content/docs/relicensing_long_tail.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Relicensing Long Tail
+bookToc: true
+---
+
+# Long tail of individuals and corporations without a relicensing agreement yet
+
+The LLVM relicensing effort is currently at the stage where we collect
+agreements from individuals and corporations owning the copyright of
+contributions before 19th of January 2019.
+
+We already have agreements in place for the far majority of contributions.  We
+still have a long tail of both individuals and corporations we'd like to get
+agreements from.
+
+The list of individuals and corporations we'd still like to get agreements from
+is regularly updated in a
+[spreadsheet](https://docs.google.com/spreadsheets/d/18_0Hog_eSwES8lKwf7WJal3yBwwcYfvPu1yCfZnTcek/edit?usp=sharing).
+Please have a look to see if you know any of them and could help with reaching
+out to them, asking them to agree with relicensing the contributions they own
+the copyright on.
+
+## Individuals
+
+When reaching out to an individual, please ask them to complete [this web
+form](https://goo.gl/forms/X4HiyYRcRHOnTSvC3). Part of that form will prompt
+them with a DocuSign agreement that they can sign online to cover anything they
+personally contributed. The web form will also collect any companies or academic
+institutions that may own rights to some of their contributions so that we can
+cover them with the corporate agreement below.
```

Comment:
Would it make sense to allow email for people who can't use docusign?

Reply:
That's a great suggestion - I've added text now to indicate we'll also accept an email for people who can't use docusign.